pkg/orchestrator/scheduler/batch_service_job.go

```diff
@@ -3,45 +3,42 @@ package scheduler
 import (
```

Comment:
Should probably rename the test `batch_job_test.go` as well

Reply:
actually `batch_job_test.go` only tests batch jobs against the shared scheduler, and `service_job_test.go` only tests service jobs against the same scheduler. I tried created a common test suite between both that has the common test scenarios and then implement the specific test scenarios under `batch_job_test` and `service_job_test`, but that didn't work well as the schedulers treat `completed` executions differently based on the job type and most test scenarios were slightly different